lib/npg_pipeline/function/status.pm

```diff
@@ -77,6 +81,9 @@ npg_pipeline::function::status
 =head1 DESCRIPTION
 
 Launches a job for saving run and lane statuses to a file.
+If saving of statuses to the database is not disabled (C<--no_db_status_update>
```

Comment:
Too complex/passive. Legibility would be easier with something like "The job will also save statuses to the tracking database unless --no_db_status_update or --local options are used"